### PR TITLE
Update tracking.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Per MS style guide
* Avoid future tense
* drop `()` from methods

[Use bigger ideas, fewer words](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#use-bigger-ideas-fewer-words) : Our modern design hinges on crisp minimalism. Shorter is always better.
[Short sentences and fragments are easier to scan and read.](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human#a-focus-on-the-customer) Prune every excess word.

Contributes to https://github.com/dotnet/AspNetCore.Docs/issues/23572

### CODE REVIEW

File: entity-framework/core/querying/tracking.md [hunk 1]
```diff
@@ -7,37 +7,41 @@ uid: core/querying/tracking
...
 
-When the results are returned in a tracking query, EF Core will check if the entity is already in the context. If EF Core finds an existing entity, then the same instance is returned. EF Core won't overwrite current and original values of the entity's properties in the entry with the database values. If the entity isn't found in the context, then EF Core will create a new entity instance and attach it to the context. Query results don't contain any entity which is added to the context but not yet saved to the database.
+When the results are returned in a tracking query, EF Core checks if the entity is already in the context. If EF Core finds an existing entity, then the same instance is returned, which can potential use less memory and be faster than a no-tracking query. EF Core doesn't overwrite current and original values of the entity's properties in the entry with the database values. If the entity isn't found in the context, EF Core creates a new entity instance and attaches it to the context. Query results don't contain any entity which is added to the context but not yet saved to the database.
```
Comment (Rick-Anderson) [inline entity-framework/core/querying/tracking.md:23]:
```suggestion
When the results are returned in a tracking query, EF Core checks if the entity is already in the context. If EF Core finds an existing entity, then the same instance is returned, which can potentially use less memory and be faster than a no-tracking query. EF Core doesn't overwrite current and original values of the entity's properties in the entry with the database values. If the entity isn't found in the context, EF Core creates a new entity instance and attaches it to the context. Query results don't contain any entity which is added to the context but not yet saved to the database.
```

File: entity-framework/core/querying/tracking.md [hunk 2]
```diff
@@ -7,37 +7,41 @@ uid: core/querying/tracking
...
 
 ## No-tracking queries
 
-No tracking queries are useful when the results are used in a read-only scenario. They're quicker to execute because there's no need to set up the change tracking information. If you don't need to update the entities retrieved from the database, then a no-tracking query should be used. You can swap an individual query to be no-tracking. No tracking query will also give you results based on what is in the database disregarding any local changes or added entities.
+No tracking queries are useful when the results are used in a read-only scenario. They're generally quicker to execute because there's no need to set up the change tracking information. If the entities retrieved from the database don't need to be updated, then a no-tracking query should be used. An individual query can be swapped to be no-tracking. A no-tracking query also give results based on what's in the database disregarding any local changes or added entities.
```
Comment (Rick-Anderson) [inline entity-framework/core/querying/tracking.md:27]:
```suggestion
No-tracking queries are useful when the results are used in a read-only scenario. They're generally quicker to execute because there's no need to set up the change tracking information. If the entities retrieved from the database don't need to be updated, then a no-tracking query should be used. An individual query can be swapped to be no-tracking. A no-tracking query also give results based on what's in the database disregarding any local changes or added entities.
```

File: entity-framework/core/querying/tracking.md [hunk 3]
```diff
@@ -7,37 +7,41 @@ uid: core/querying/tracking
...
+Both of the tracking and no-tracking can be combined in the same query. That is, you can have a no-tracking query, which  does identity resolution in the results. Just like `AsNoTracking` queryable operator, we've added another operator <xref:Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.AsNoTrackingWithIdentityResolution%60%601(System.Linq.IQueryable{%60%600})>. There's also associated entry added in the <xref:Microsoft.EntityFrameworkCore.QueryTrackingBehavior> enum. When the query to use identity resolution is configured with no tracking, a stand-alone change tracker is used in the background when generating query results so each instance is materialized only once. Since this change tracker is different from the one in the context, the results are not tracked by the context. After the query is enumerated fully, the change tracker goes out of scope and garbage collected as required.
```
Comment (Rick-Anderson) [inline entity-framework/core/querying/tracking.md:44]:
```suggestion
Tracking and no-tracking can be combined in the same query. That is, you can have a no-tracking query, which  does identity resolution in the results. Just like `AsNoTracking` queryable operator, we've added another operator <xref:Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.AsNoTrackingWithIdentityResolution%60%601(System.Linq.IQueryable{%60%600})>. There's also associated entry added in the <xref:Microsoft.EntityFrameworkCore.QueryTrackingBehavior> enum. When the query to use identity resolution is configured with no tracking, a stand-alone change tracker is used in the background when generating query results so each instance is materialized only once. Since this change tracker is different from the one in the context, the results are not tracked by the context. After the query is enumerated fully, the change tracker goes out of scope and garbage collected as required.
```

File: entity-framework/core/querying/tracking.md [hunk 4]
```diff
@@ -47,7 +51,7 @@ If you find yourself changing the tracking behavior for many queries, you may wa
 
 [!code-csharp[Main](../../../samples/core/Querying/Tracking/NonTrackingBloggingContext.cs?name=OnConfiguring&highlight=5)]
 
-This makes all your queries non-tracking by default. You can still add `AsTracking()` to make specific queries tracking.
+This makes all your queries no-tracking by default. You can still add `AsTracking()` to make specific queries tracking.
```
Comment (Rick-Anderson) [inline entity-framework/core/querying/tracking.md:54]:
```suggestion
This makes all your queries no-tracking by default. You can still add `AsTracking` to make specific queries tracking.
```

File: entity-framework/core/querying/tracking.md [hunk 5]
```diff
@@ -7,37 +7,41 @@ uid: core/querying/tracking
...
 
 ## No-tracking queries
 
-No tracking queries are useful when the results are used in a read-only scenario. They're quicker to execute because there's no need to set up the change tracking information. If you don't need to update the entities retrieved from the database, then a no-tracking query should be used. You can swap an individual query to be no-tracking. No tracking query will also give you results based on what is in the database disregarding any local changes or added entities.
+No-tracking queries are useful when the results are used in a read-only scenario. They're generally quicker to execute because there's no need to set up the change tracking information. If the entities retrieved from the database don't need to be updated, then a no-tracking query should be used. An individual query can be swapped to be no-tracking. A no-tracking query also give results based on what's in the database disregarding any local changes or added entities.
```
Comment (roji) [inline entity-framework/core/querying/tracking.md:27]:
```suggestion
No-tracking queries are useful when the results are used in a read-only scenario. They're generally quicker to execute because there's no need to set up the change tracking information. If the entities retrieved from the database don't need to be updated, then a no-tracking query should be used. An individual query can be set to be no-tracking. A no-tracking query also give results based on what's in the database disregarding any local changes or added entities.
```

File: entity-framework/core/querying/tracking.md [hunk 6]
```diff
@@ -7,37 +7,41 @@ uid: core/querying/tracking
...
+Tracking and no-tracking can be combined in the same query. That is, you can have a no-tracking query, which  does identity resolution in the results. Just like `AsNoTracking` queryable operator, we've added another operator <xref:Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.AsNoTrackingWithIdentityResolution%60%601(System.Linq.IQueryable{%60%600})>. There's also associated entry added in the <xref:Microsoft.EntityFrameworkCore.QueryTrackingBehavior> enum. When the query to use identity resolution is configured with no tracking, a stand-alone change tracker is used in the background when generating query results so each instance is materialized only once. Since this change tracker is different from the one in the context, the results are not tracked by the context. After the query is enumerated fully, the change tracker goes out of scope and garbage collected as required.
```
Comment (roji) [inline entity-framework/core/querying/tracking.md:44]:
```suggestion
Tracking and no-tracking can be combined in the same query. That is, you can have a no-tracking query, which does identity resolution in the results. Just like `AsNoTracking` queryable operator, we've added another operator <xref:Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.AsNoTrackingWithIdentityResolution%60%601(System.Linq.IQueryable{%60%600})>. There's also associated entry added in the <xref:Microsoft.EntityFrameworkCore.QueryTrackingBehavior> enum. When the query to use identity resolution is configured with no tracking, a stand-alone change tracker is used in the background when generating query results so each instance is materialized only once. Since this change tracker is different from the one in the context, the results are not tracked by the context. After the query is enumerated fully, the change tracker goes out of scope and garbage collected as required.
```

Comment (roji) [inline entity-framework/core/querying/tracking.md:44]: We probably want to make other method names (AsNoTracking, AsTracking) xref links as well

**Review by roji — APPROVED** [review]
LGTM but will let @ajcvickers approve as well.

Re changing 2nd-person ("you can do this") to impersonal ("this can be done"), I thought a more personal tone was OK/encouraged, we certainly use 2nd-person in many places in our docs.

File: entity-framework/core/querying/tracking.md (hunk 6 above)
Comment (Rick-Anderson) [reply entity-framework/core/querying/tracking.md]: > We probably want to make other method names (AsNoTracking, AsTracking) xref links as well

@roji  please review [this commit](https://github.com/dotnet/EntityFramework.Docs/pull/4298/commits/65f734987569747c757ee2c5f5b3f247085fdce1)

File: entity-framework/core/querying/tracking.md [hunk 7]
```diff
@@ -2,42 +2,46 @@
...
 # Tracking vs. No-Tracking Queries
 
-Tracking behavior controls if Entity Framework Core will keep information about an entity instance in its change tracker. If an entity is tracked, any changes detected in the entity will be persisted to the database during `SaveChanges()`. EF Core will also fix up navigation properties between the entities in a tracking query result and the entities that are in the change tracker.
+Tracking behavior controls if Entity Framework Core keeps information about an entity instance in its change tracker. If an entity is tracked, any changes detected in the entity are persisted to the database during `SaveChanges`. EF Core also fixes up navigation properties between the entities in a tracking query result and the entities that are in the change tracker.
 
 > [!NOTE]
 > [Keyless entity types](xref:core/modeling/keyless-entity-types) are never tracked. Wherever this article mentions entity types, it refers to entity types which have a key defined.
 
-> [!TIP]
-> You can view this article's [sample](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Querying/Tracking) on GitHub.
```
Comment (ajcvickers) [inline entity-framework/core/querying/tracking.md:16]: Why this change?

**Review by ajcvickers — CHANGES_REQUESTED** [review]
PR not passing checks.

File: entity-framework/core/querying/tracking.md (hunk 7 above)
Comment (Rick-Anderson) [reply entity-framework/core/querying/tracking.md:16]: IMAO, too much bling can be distracting. I can revert.

**Comment by Rick-Anderson** [comment]
> PR not passing checks.

Nothing to do with my changes

 Error: entity-framework/core/querying/tracking.md:82:1 MD004/ul-style Unordered list style [Expected: asterisk; Actual: dash]

I'll fix these.

File: entity-framework/core/querying/tracking.md (hunk 7 above)
Comment (ajcvickers) [reply entity-framework/core/querying/tracking.md:16]: Yes, please.

File: entity-framework/core/querying/tracking.md [hunk 8]
```diff
@@ -2,42 +2,46 @@
...
+You can view this article's [sample](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Querying/Tracking) on GitHub.
```
Comment (Rick-Anderson) [inline entity-framework/core/querying/tracking.md:15]:
```suggestion
> [!TIP]
> You can view this article's [sample](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Querying/Tracking) on GitHub.
```

**Review by ajcvickers — APPROVED** [review]
